packages/widget-welcome/src/extension.ts

```diff
@@ -75,6 +76,7 @@ class StateManager extends ExtensionManager<State & Events, Configuration> {
   async fetchData () {
     const url = `${this.backendUrl}/getTricks`
     this._channel.appendLine(`Fetching ${url}`)
+    axiosRetry(Axios, { retries: 10, retryDelay: axiosRetry.exponentialDelay })
```

Comment:
I think we can move this up to the top of the file as we only need to call it once.